app/policies/app_integration_policy.rb

```diff
@@ -0,0 +1,18 @@
+# Forem apps AppIntegration are not meant to be modified by creators
+class AppIntegrationPolicy < ApplicationPolicy
+  def create?
+    !@record.forem_app?
```

Comment:
**nitpick (non-blocking):** I think this would actually read better if not-negated, i.e. 

```suggestion
    @record.forem_app.nil?
```

That said even with the top-level comment I'm not sure I 100% understand the purpose of this.

Reply:
Yes, much better, thank you!